### PR TITLE
[access-21] Added Users Info on Registration

### PR DESCRIPTION
**Pull request by jandyl-casas**
[users_info] table needs to add new columns:
- created_at 
- updated_at
this is the same attributes on what existed on [users] table.

N.B. This is another issue. Registration will have an error if this two columns will not be added.

### CODE REVIEW

File: resources/views/auth/register.blade.php [hunk 1]
```diff
@@ -47,6 +47,76 @@
...
+                            <div class="col-md-6">
+                                <input id="firstname" type="text" class="form-control" name="firstname" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="middlename" class="col-md-4 control-label">Middle name</label>
+
+                            <div class="col-md-6">
+                                <input id="middlename" type="text" class="form-control" name="middlename" required>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: middlename should not be required

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -47,6 +47,76 @@
                         </div>
 
                         <div class="form-group">
+                            <label for="firstname" class="col-md-4 control-label">First name</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: use "First Name"

File: resources/views/auth/register.blade.php [hunk 3]
```diff
@@ -47,6 +47,76 @@
...
+
+                            <div class="col-md-6">
+                                <input id="firstname" type="text" class="form-control" name="firstname" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="middlename" class="col-md-4 control-label">Middle name</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: use "Middle Name"

File: resources/views/auth/register.blade.php [hunk 4]
```diff
@@ -47,6 +47,76 @@
...
+
+                            <div class="col-md-6">
+                                <input id="middlename" type="text" class="form-control" name="middlename" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="lastname" class="col-md-4 control-label">Last name</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: use "Last Name"

File: resources/views/auth/register.blade.php [hunk 5]
```diff
@@ -47,6 +47,76 @@
...
+
+                            <div class="col-md-6">
+                                <input id="lastname" type="text" class="form-control" name="lastname" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                            <label for="email" class="col-md-4 control-label">Email address</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: remove "address"

File: resources/views/auth/register.blade.php [hunk 6]
```diff
@@ -47,6 +47,76 @@
...
+                                @if ($errors->has('email'))
+                                    <span class="help-block">
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="telnum" class="col-md-4 control-label">Telephone number</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: use "Telephone No."

File: resources/views/auth/register.blade.php [hunk 7]
```diff
@@ -47,6 +47,76 @@
...
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="telloc" class="col-md-4 control-label">Telephone local number</label>
+
+                            <div class="col-md-6">
+                                <input id="telloc" type="text" class="form-control" name="telloc" required>
+                            </div>
+                        </div>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: I guess this is unnecessary. Much better if we just remove the field "Telephone local number".

File: resources/views/auth/register.blade.php [hunk 8]
```diff
@@ -47,6 +47,76 @@
...
+
+                        <div class="form-group">
+                            <label for="mobilenum" class="col-md-4 control-label">Mobile number</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: use "Mobile"

File: resources/views/auth/register.blade.php [hunk 9]
```diff
@@ -47,6 +47,76 @@
...
+
+                            <div class="col-md-6">
+                                <input id="mobilenum" type="text" class="form-control" name="mobilenum" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="mobileloc" class="col-md-4 control-label">Mobile local number</label>
```
Comment (rosoarv) [inline resources/views/auth/register.blade.php]: same with telephone local number

File: app/UsersInfo.php [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    * @var string
+    */
+    protected $primaryKey = 'users_info_id';
+
+    /**
+     * Attributes that should be mass-assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['firstname', 'lastname', 'middlename', 'email', 'tel_loc', 'tel_num', 'mobile_loc', 'mobile_num', 'user_id'];
```
Comment (rosoarv) [inline app/UsersInfo.php]: update based on the updated fields on the form

**Review by rosoarv — CHANGES_REQUESTED** [review]

**Comment by jandyl-casas** [comment]
[users] table needs to delete the following columns which is no longer needed:
    - tel_loc
    - mobile_loc
N.B. This is another issue. Registration will have an error if this two columns will not be deleted.

**Review by rosoarv — APPROVED** [review]

**Comment by rosoarv** [comment]
@jandyl-casas please merge to master. thanks!

**Pull request merged**